### PR TITLE
Screen: Confirmed New Infections - string update (EXPOSUREAPP-4510)

### PR DESCRIPTION
**Pull request by jurajkusnier**
PR for translations

### CODE REVIEW

**Review by ralfgehrer — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1280,13 +1280,34 @@
...
     <!-- XHED: Title for incident statistics card -->
-    <string name="statistics_card_incidence_title">7-Tage-Inzidenz</string>
-
-    <!-- XTXT: Label in statistics card that is displayed above the incidence value-->
-    <string name="statistics_card_incidence_value_label">Bis gestern</string>
-
+    <string name="statistics_card_incidence_title">"7-Tage-Inzidenz"</string>
     <!-- XTXT: Description in statistics card that is displayed below the incidence value-->
-    <string name="statistics_card_incidence_value_description">bestätigte Neuinfektionen je 100.000 Einwohner</string>
+    <string name="statistics_card_incidence_value_description">"bestätigte Neuinfektionen je 100.000 Einwohner"</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1285]:
```suggestion
    <string name="statistics_card_incidence_value_description">"Bestätigte Neuinfektionen je 100.000 Einwohner"</string>
```

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1280,13 +1280,34 @@
...
+
+    <!-- Infections statistics card -->
+    <!-- XHED: Title for infections statistics card -->
+    <string name="statistics_card_infections_title">"Bestätigte Neuinfektionen"</string>
+    <!-- XTXT: Label in infections statistics card that is displayed above the secondary value-->
+    <string name="statistics_card_infections_secondary_label">"7-Tage-Mittelwert"</string>
+    <!-- XTXT: Label in infections statistics card that is displayed above the tertiary value-->
+    <string name="statistics_card_infections_tertiary_label">"Gesamt"</string>
+    <!-- XTXT: Text displayed in the bottom of infections statistics card-->
+    <string name="statistics_card_infections_bottom_text">"bundesweite Neuinfektionen"</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1295]:
```suggestion
    <string name="statistics_card_infections_bottom_text">"Bundesweite Neuinfektionen"</string>
```

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -1280,13 +1280,34 @@
...
+
+    <!-- Incidence statistics card -->
+    <!-- XHED: Title for submission statistics card -->
+    <string name="statistics_card_submission_title">"Warnende Personen"</string>
+    <!-- XTXT: Text displayed in the bottom of submission statistics card-->
+    <string name="statistics_card_submission_bottom_text">"bundesweit warnende Personen"</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1301]:
```suggestion
    <string name="statistics_card_submission_bottom_text">"Bundesweit warnende Personen"</string>
```

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1284,13 +1284,34 @@
...
     <!-- XHED: Title for incident statistics card -->
-    <string name="statistics_card_incidence_title">7-Tage-Inzidenz</string>
-
-    <!-- XTXT: Label in statistics card that is displayed above the incidence value-->
-    <string name="statistics_card_incidence_value_label">Bis gestern</string>
-
+    <string name="statistics_card_incidence_title">"7-Tage-Inzidenz"</string>
     <!-- XTXT: Description in statistics card that is displayed below the incidence value-->
-    <string name="statistics_card_incidence_value_description">bestätigte Neuinfektionen je 100.000 Einwohner</string>
+    <string name="statistics_card_incidence_value_description">"bestätigte Neuinfektionen je 100.000 Einwohner"</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values/strings.xml:1289]:
```suggestion
    <string name="statistics_card_incidence_value_description">"Bestätigte Neuinfektionen je 100.000 Einwohner"</string>
```

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1284,13 +1284,34 @@
...
+
+    <!-- Infections statistics card -->
+    <!-- XHED: Title for infections statistics card -->
+    <string name="statistics_card_infections_title">"Bestätigte Neuinfektionen"</string>
+    <!-- XTXT: Label in infections statistics card that is displayed above the secondary value-->
+    <string name="statistics_card_infections_secondary_label">"7-Tage-Mittelwert"</string>
+    <!-- XTXT: Label in infections statistics card that is displayed above the tertiary value-->
+    <string name="statistics_card_infections_tertiary_label">"Gesamt"</string>
+    <!-- XTXT: Text displayed in the bottom of infections statistics card-->
+    <string name="statistics_card_infections_bottom_text">"bundesweite Neuinfektionen"</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values/strings.xml:1299]:
```suggestion
    <string name="statistics_card_infections_bottom_text">"Bundesweite Neuinfektionen"</string>
```

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1284,13 +1284,34 @@
...
+
+    <!-- Incidence statistics card -->
+    <!-- XHED: Title for submission statistics card -->
+    <string name="statistics_card_submission_title">"Warnende Personen"</string>
+    <!-- XTXT: Text displayed in the bottom of submission statistics card-->
+    <string name="statistics_card_submission_bottom_text">"bundesweit warnende Personen"</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values/strings.xml:1305]:
```suggestion
    <string name="statistics_card_submission_bottom_text">"Bundesweit warnende Personen"</string>
```

**Review by SabineLoss — COMMENTED** [review]
@janetback - I can't see a reason why these strings are lowercase, but please double check.

**Comment by jurajkusnier** [comment]
> @janetback - I can't see a reason why these strings are lowercase, but please double check.

All strings are directly copied from the design in Figma.

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (ralfgehrer) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:1285]: FYI: I asked for clarification as it mismatches Figma

Comment (ralfgehrer) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:1285]: Sabine agreed to keep the small caps

**Pull request merged**